apps/src/music/views/Callouts.tsx

```diff
@@ -71,12 +71,11 @@ const availableCallouts: AvailableCallouts = {
   'trigger-button-1': {selector: `#${Triggers[0].id}`},
   'toolbox-first-row': {selector: '.blocklyTreeRow'},
   'flyout-first-block': {
-    selector:
-      '.blocklyFlyout:not([style*="display: none;"]) .blocklyDraggable:nth-of-type(1)',
```

Comment:
This only worked when there was more than one matching `.blocklyDraggable`.  `document.querySelector` returns the first match, so the `:nth-of-type(1)` was redundant.